### DISCUSSION TITLE
Any updates guys?

### DISCUSSION THREAD

**Discussion by neilpeel13** [mRemoteNG/mRemoteNG#2645] [General]
Just wondering if there have been any update or news of a release?
We were advised a release for last Christmas, but that never arrived. Should we give up hope of seeing anything on this project??

TBH I'd have preferred amore incremental release, fixing bugs gradually, than a long time between releases. But hey ho. Fingers crossed there will be announcement at some point soon.

**Comment by Kvarkas** [comment]
hey hey, delay with refactoring and security patch, not easy to fix, need to recode a lot of parts, working on that then have free time

**Reply by neilpeel13** [reply]
Thanks! I don't suppose you have a rough ETA? I mean, this year or next?  :)
